kiwixbuild/dependencies/base.py

```diff
@@ -72,8 +72,11 @@ def command(self, name, function, *args):
             context._finalise()
             print("OK")
             return ret
-        except SkipCommand:
-            print("SKIP")
+        except SkipCommand as e:
+            message = "SKIP"
+            if e.message:
+                message += " : " + e.message
+            print(message)
```

Comment:
It would be better that `SkipCommand` handles the message formatting itself (using `__str__` method).
This would allow to simply have :
```
except SkipCommand as e:
    print(e)
```

Reply:
done